.travis.yml

```diff
@@ -18,13 +11,13 @@ before-script:
 
 script:
   - docker run my-test-image tests/test_viewmodel.py
-  - docker run my-test-image tests/test_integration.py
-  - >
-     docker run
-     -e API_KEY=$API_KEY
-     -e TOKEN=$TOKEN
-     -e BOARD_ID=$BOARD_ID
-     my-test-image tests/test_e2e.py
+#  - docker run my-test-image tests/test_integration.py
+#  - >
+#     docker run
+#     -e API_KEY=$API_KEY
+#     -e TOKEN=$TOKEN
+#    -e BOARD_ID=$BOARD_ID
+#     my-test-image tests/test_e2e.py
```

Comment:
It would be nice to switch the integration/end-to-end tests back on since you've done 95% of the work! I'll leave some suggestions below for that